src/components/NewDraftWizard/WizardSteps/WizardCreateFolderStep.tsx

```diff
@@ -152,23 +152,25 @@ const CreateFolderForm = ({ createFolderFormRef }: { createFolderFormRef: Create
           )}
           rules={{ required: true, validate: { description: value => value.length > 0 } }}
         />
+
+        <Grid container spacing={2} className={classes.typeOfSubmissionRow}>
+          <Grid item>
+            <div className={classes.typeOfSubmissionLabel}>Type of submission</div>
+          </Grid>
+          <Grid item xs={6}>
+            <FormControl>
+              <RadioGroup name="submission-type-selection">
+                <FormControlLabel value="FEGA" control={<Radio />} label="FEGA" />
+                <FormControlLabel value="BigPicture" control={<Radio />} label="BigPicture" />
```

Comment:
`BigPicture` might be a bit too early to include here, as it requires additional form/JSON schemas in the backend. By default we need to have a `General` type maybe

Reply:
I've added default value for "General" and disabled placeholder for "FEGA"